.github/workflows/ci.yml

```diff
@@ -145,8 +134,63 @@ jobs:
       - name: Test
         run: ./phpunit
         shell: bash
+
+  test-reset-database-with-migration:
+    name: Test migration - D:${{ matrix.database }}, ${{ matrix.use-dama == 1 && ' (dama)' || '' }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        database: [ mysql, pgsql, sqlite ]
+        use-dama: [ 0, 1 ]
+        exclude:
+          - database: sqlite
+            use-dama: 0
```

Comment:
there is a problem with migrate + sqlite + "no dama" :
because we're calling `dropAndResetDatabase()` which does not actually drop the database for sqlite:
https://github.com/zenstruck/foundry/blob/2.x/src/ORM/AbstractORMPersistenceStrategy.php#L126-L129

I'm wondering if we should not remove the actual file `sqlite.db` here, but I'm not sure we can access it easily :thinking:

Reply:
ok, I've actually removed the .db file, and it fixes the problem